crates/noirc_evaluator/src/brillig/brillig_gen/brillig_directive.rs

```diff
@@ -0,0 +1,16 @@
+use acvm::acir::brillig_vm::{BinaryFieldOp, Opcode as BrilligOpcode, RegisterIndex, Value};
+
+/// Generates brillig bytecode which computes the inverse of its input if not null, and zero else.
+pub(crate) fn directive_invert() -> Vec<BrilligOpcode> {
+    vec![
+        BrilligOpcode::JumpIfNot { condition: RegisterIndex::from(0), location: 3 },
```

Comment:
Instead of inlining in the vector, can you outline it and document it. An example using pseudocode:

```rust
pub(crate) fn directive_invert() -> Vec<BrilligOpcode> {
 //  The code being generated is the following:
 // fn invert(x : Field) -> Field {
 //    1/ x
 // }

let input = RegisterIndex::from(0);

// If the input is zero, then we jump to the stop opcode
let stop_location = 3;
let jmp_if_not = BrilligOpcode::JumpIfNot { condition: input, location: stop_location };
...
}
```